packages/frontend/scss/misc/_keyboard_hint_box.scss

```diff
@@ -47,29 +47,31 @@
 }
 
 .keyboard-hint-cheatsheet-dialog {
+  width: 80% !important;
+  max-width: 560px !important;
   .keybinding {
-    display: block;
-    flex-grow: 1;
     font-size: 0.84em;
-    margin-bottom: 0;
+  }
+  h4 .keybinding {
+    flex: none;
+    margin-left: 20px;
+  }
+  .dialog-body {
+    padding-bottom: 0;
   }
   .keyboard-hint-dialog-body {
     display: flex;
     flex-wrap: wrap;
-    flex-direction: column;
```

Comment:
Don't you think the column order is better? To me it seems intuitive to read it top-to-bottom. And there are no sections that are extra wide:

![image](https://github.com/user-attachments/assets/390184ce-40ac-412f-9112-01e1c67e0f60)

Reply:
If a set that it will result in just one long column which is definitely not better. Or we need 2 columns in the template. But tbh that (and this discussions about edge cases in a rarely used screen) seems a bit overdone for me.

But feel free to add a commit with better styling that fix your concerns :-)

Reply:
Oh. I get it. We have column order on `main`.